app/static/css/_form.scss

```diff
@@ -92,31 +98,31 @@
 .date-picker {
     justify-content: flex-start;
     margin: 0;
-    padding: 0.5rem 0;
 }
 
 .date-picker__group {
     flex: 0 1 auto;
+    padding: 0.5rem 0;
 }
 
-
 .loading {
     width: 100%; 
     margin:  auto;
     display: none;
     background-color: constants.$soft-white;
+    opacity: 90%;
```

Comment:
This makes the loading spinner take up the whole page and be slightly transparent (so people feel less like they have lost their place, but more like something is happening)